tests/testthat/test-data-combined.R

```diff
@@ -128,15 +128,61 @@ test_that("setting groups fails when group specification is invalid", {
   myCombDat <- DataCombined$new()
   myCombDat$addDataSets(dataSet)
 
-  expect_error(myCombDat$setGroups(c(2, 4)))
-  expect_error(myCombDat$setGroups(list("x" = 2, "y" = 4)))
+  expect_error(
+    myCombDat$setGroups(c(2, 4)),
+    "You need to provide a named list with at least one valid grouping."
```

Comment:
This is where using a constant file for strings would be much easier and future proof. Right now, we have "text" spread out everywhere. Changing a typo or modifying the text should (arguably) not break a test.

Reply:
Let's track this in https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/issues/76

Reply:
I don't think package specific messages should be in utils. They should be in the lowest package possible in the hierarchy